### ISSUE TITLE
Warriors are to aggresive

### ISSUE THREAD

**Issue by jaschaio** [jsettlers/settlers-remake#690]
I am not sure if warriors worked like that in the original game, but it is very hard to move warriors anywhere else when enemies are near. You have to click all the time in the opposite direction because they will keep returning and its almost impossible to make them follow any orders. This is quite annoying as you basically have to savekeep your warriors and can't get anything else done in that time.

**Comment by DerKrawallkeks** [comment]
Yes you're right. In the original game, they used to listen to your commands even without ctrl and even if there's an enemy around. Only when they spot someone new or if triggered by something or wait too long, they'll try to fight. Which could always be avoided by ctrl. 

But I think there's already an issue opened about that..not sure. Someone might be able to move this issue.

Would be cool if it was solved, cause it's one of the biggest issues with playability.

**Comment by jaschaio** [comment]
Yeah #661 is related and now I searched a bit more and found #506. After playing some more time I agree that this is one of the biggest issues with playability as it forces you to micromanage soldiers which can take quite some time that you can't invest anywhere else.

So CTRL + Click on Computers would solve it. And if we are already talking about it, having the ability to add various groups with ALT + #NUMBER was a great future of the original game as well.

**Comment by andreas-eberle** [comment]
Yes, we have to work on this. I'll close this in favor of #506.

**Comment by PaweX** [comment]
> So CTRL + Click on Computers would solve it. And if we are already talking about it, having the ability to add various groups with ALT + #NUMBER was a great future of the original game as well.
Opposite!
'ALT+RMB' for forcing to move and 'CTRL+Numbers1-9' to create a group. But see this first: #760

> in the Settlers III we can just command our soldiers to move and they stop fighting and go were we send them.

**Comment by deisme01** [comment]
the game would be absolutely playable if the behaviour of the soldiers was different.
it's impossible to get your soldiers away from an enemy soldier not even with ALT+RMB'.

please keep working on it!
thumbs up!!!!!!!!!

**Comment by paulwedeck** [comment]
@deisme01 There already is a pr for this #709
